services/src/test/java/org/keycloak/authentication/authenticators/x509/CertificateValidatorTest.java

```diff
@@ -0,0 +1,157 @@
+package org.keycloak.authentication.authenticators.x509;
```

Comment:
I would much prefer if this was converted to a functional test by extending one of the tests in testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/x509. Would that be achievable?

Reply:
Tried it but unfortunately I got trouble in getting it running. Could not determine what exactly the problem was and how to debug. Currently I am not having enough time to check this out in detail...

Reply:
was not able to get the test running and I am not getting any answers in the mailing list...
https://groups.google.com/forum/#!topic/keycloak-dev/MI20c_zKI1Q